cmd/server-startup-msg.go

```diff
@@ -154,7 +154,7 @@ func printServerCommonMsg(apiEndpoints []string) {
 
 // Prints startup message for Object API acces, prints link to our SDK documentation.
 func printObjectAPIMsg() {
-	logger.Info(color.Blue("\nDocumentation: ") + "https://docs.min.io")
+	logger.Info(color.Blue("\nDocumentation: ") + "https://min.io/docs/minio/linux/developers/minio-drivers.html")
```

Comment:
Hm - for this we can probably just point to the linux root rather than drivers.

I get the idea though. If/when we have our full developer site, it would make sense to have a dedicated line item for 'SDK Documentation' or something.

Reply:
Changed to root Linux docs.